src/c++/perf_analyzer/concurrency_worker.cc

```diff
@@ -364,4 +310,13 @@ ConcurrencyWorker::CompleteOngoingSequences()
   return cb::Error::Success;
 }
 
+void
+ConcurrencyWorker::WaitForOngoingRequests()
+{
+  {
```

Comment:
Don't need this extra scope here. You aren't locking anything.

Reply:
fixed.